/work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:283: FutureWarning: `torch.cuda.amp.autocast(args...)` is deprecated. Please use `torch.amp.autocast('cuda', args...)` instead.
  @autocast(enabled=False)
/work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:418: FutureWarning: `torch.cuda.amp.autocast(args...)` is deprecated. Please use `torch.amp.autocast('cuda', args...)` instead.
  @autocast(enabled=False)
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/torch/_dynamo/variables/functions.py:663: UserWarning: Graph break due to unsupported builtin torch._C._to_dlpack. This function is either a Python builtin (e.g. _warnings.warn) or a third-party C/C++ Python extension (perhaps created with pybind). If it is a Python builtin, please file an issue on GitHub so the PyTorch team can add support for it and see the next case for a workaround. If it is a third-party C/C++ Python extension, please either wrap it into a PyTorch-understood custom operator (see https://pytorch.org/tutorials/advanced/custom_ops_landing_page.html for more details) or, if it is traceable, use torch.compiler.allow_in_graph.
  torch._dynamo.utils.warn_once(msg)
V0421 01:20:14.206271 47802126279936 torch/_dynamo/guards.py:2611] [4/1] [__recompiles] Recompiling function create_unitgraph_from_coo in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1254
V0421 01:20:14.206271 47802126279936 torch/_dynamo/guards.py:2611] [4/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:14.206271 47802126279936 torch/_dynamo/guards.py:2611] [4/1] [__recompiles]     - tensor 'L['row']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 01:20:14.223474 47802126279936 torch/_dynamo/guards.py:2611] [5/1] [__recompiles] Recompiling function to_dgl_nd in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/__init__.py:141
V0421 01:20:14.223474 47802126279936 torch/_dynamo/guards.py:2611] [5/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:14.223474 47802126279936 torch/_dynamo/guards.py:2611] [5/1] [__recompiles]     - tensor 'L['data']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 01:20:14.232318 47802126279936 torch/_dynamo/guards.py:2611] [6/1] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:432
V0421 01:20:14.232318 47802126279936 torch/_dynamo/guards.py:2611] [6/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:14.232318 47802126279936 torch/_dynamo/guards.py:2611] [6/1] [__recompiles]     - tensor 'L['data']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 01:20:14.260746 47802126279936 torch/_dynamo/guards.py:2611] [8/1] [__recompiles] Recompiling function torch_dynamo_resume_in_create_unitgraph_from_coo_at_1296 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1296
V0421 01:20:14.260746 47802126279936 torch/_dynamo/guards.py:2611] [8/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:14.260746 47802126279936 torch/_dynamo/guards.py:2611] [8/1] [__recompiles]     - tensor 'L['col']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
W0421 01:20:24.207666 47802126279936 torch/_dynamo/variables/tensor.py:715] [30/0] Graph break from `Tensor.item()`, consider setting:
W0421 01:20:24.207666 47802126279936 torch/_dynamo/variables/tensor.py:715] [30/0]     torch._dynamo.config.capture_scalar_outputs = True
W0421 01:20:24.207666 47802126279936 torch/_dynamo/variables/tensor.py:715] [30/0] or:
W0421 01:20:24.207666 47802126279936 torch/_dynamo/variables/tensor.py:715] [30/0]     env TORCHDYNAMO_CAPTURE_SCALAR_OUTPUTS=1
W0421 01:20:24.207666 47802126279936 torch/_dynamo/variables/tensor.py:715] [30/0] to include these operations in the captured graph.
W0421 01:20:24.207666 47802126279936 torch/_dynamo/variables/tensor.py:715] [30/0] 
V0421 01:20:27.343311 47802126279936 torch/_dynamo/guards.py:2611] [4/2] [__recompiles] Recompiling function create_unitgraph_from_coo in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1254
V0421 01:20:27.343311 47802126279936 torch/_dynamo/guards.py:2611] [4/2] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:27.343311 47802126279936 torch/_dynamo/guards.py:2611] [4/2] [__recompiles]     - tensor 'L['row']' size mismatch at index 0. expected 0, actual 2168
V0421 01:20:27.343311 47802126279936 torch/_dynamo/guards.py:2611] [4/2] [__recompiles]     - tensor 'L['row']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 01:20:27.371548 47802126279936 torch/_dynamo/guards.py:2611] [5/2] [__recompiles] Recompiling function to_dgl_nd in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/__init__.py:141
V0421 01:20:27.371548 47802126279936 torch/_dynamo/guards.py:2611] [5/2] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:27.371548 47802126279936 torch/_dynamo/guards.py:2611] [5/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 0, actual 2168
V0421 01:20:27.371548 47802126279936 torch/_dynamo/guards.py:2611] [5/2] [__recompiles]     - tensor 'L['data']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 01:20:27.383489 47802126279936 torch/_dynamo/guards.py:2611] [6/2] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:432
V0421 01:20:27.383489 47802126279936 torch/_dynamo/guards.py:2611] [6/2] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:27.383489 47802126279936 torch/_dynamo/guards.py:2611] [6/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 0, actual 2168
V0421 01:20:27.383489 47802126279936 torch/_dynamo/guards.py:2611] [6/2] [__recompiles]     - tensor 'L['data']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 01:20:27.422868 47802126279936 torch/_dynamo/guards.py:2611] [8/2] [__recompiles] Recompiling function torch_dynamo_resume_in_create_unitgraph_from_coo_at_1296 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1296
V0421 01:20:27.422868 47802126279936 torch/_dynamo/guards.py:2611] [8/2] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:27.422868 47802126279936 torch/_dynamo/guards.py:2611] [8/2] [__recompiles]     - tensor 'L['col']' size mismatch at index 0. expected 0, actual 2168
V0421 01:20:27.422868 47802126279936 torch/_dynamo/guards.py:2611] [8/2] [__recompiles]     - tensor 'L['col']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 01:20:27.440200 47802126279936 torch/_dynamo/guards.py:2611] [32/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_574 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:574
V0421 01:20:27.440200 47802126279936 torch/_dynamo/guards.py:2611] [32/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:27.440200 47802126279936 torch/_dynamo/guards.py:2611] [32/1] [__recompiles]     - L['___stack0'] == 1048                                      
V0421 01:20:27.660283 47802126279936 torch/_dynamo/guards.py:2611] [34/1] [__recompiles] Recompiling function cat in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:222
V0421 01:20:27.660283 47802126279936 torch/_dynamo/guards.py:2611] [34/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:27.660283 47802126279936 torch/_dynamo/guards.py:2611] [34/1] [__recompiles]     - tensor 'L['seq'][0]' size mismatch at index 0. expected 0, actual 2168
V0421 01:20:28.408630 47802126279936 torch/_dynamo/guards.py:2611] [20/1] [__recompiles] Recompiling function add_edges in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:435
V0421 01:20:28.408630 47802126279936 torch/_dynamo/guards.py:2611] [20/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:28.408630 47802126279936 torch/_dynamo/guards.py:2611] [20/1] [__recompiles]     - tensor 'L['u']' size mismatch at index 0. expected 2168, actual 564
V0421 01:20:28.430579 47802126279936 torch/_dynamo/guards.py:2611] [26/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_541 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:541
V0421 01:20:28.430579 47802126279936 torch/_dynamo/guards.py:2611] [26/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:28.430579 47802126279936 torch/_dynamo/guards.py:2611] [26/1] [__recompiles]     - tensor 'L['v']' size mismatch at index 0. expected 2168, actual 564
V0421 01:20:28.458098 47802126279936 torch/_dynamo/guards.py:2611] [27/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_542 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:542
V0421 01:20:28.458098 47802126279936 torch/_dynamo/guards.py:2611] [27/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:28.458098 47802126279936 torch/_dynamo/guards.py:2611] [27/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 2168, actual 564
V0421 01:20:28.475228 47802126279936 torch/_dynamo/guards.py:2611] [28/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_545 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:545
V0421 01:20:28.475228 47802126279936 torch/_dynamo/guards.py:2611] [28/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:28.475228 47802126279936 torch/_dynamo/guards.py:2611] [28/1] [__recompiles]     - tensor 'L['u']' size mismatch at index 0. expected 2168, actual 564
V0421 01:20:28.542668 47802126279936 torch/_dynamo/guards.py:2611] [30/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_573 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:573
V0421 01:20:28.542668 47802126279936 torch/_dynamo/guards.py:2611] [30/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:28.542668 47802126279936 torch/_dynamo/guards.py:2611] [30/1] [__recompiles]     - tensor 'L['u']' size mismatch at index 0. expected 2168, actual 564
V0421 01:20:29.003904 47802126279936 torch/_dynamo/guards.py:2611] [32/2] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_574 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:574
V0421 01:20:29.003904 47802126279936 torch/_dynamo/guards.py:2611] [32/2] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:29.003904 47802126279936 torch/_dynamo/guards.py:2611] [32/2] [__recompiles]     - tensor 'L['v']' size mismatch at index 0. expected 2168, actual 564
V0421 01:20:29.003904 47802126279936 torch/_dynamo/guards.py:2611] [32/2] [__recompiles]     - L['___stack0'] == 1048                                      
V0421 01:20:29.253228 47802126279936 torch/_dynamo/guards.py:2611] [34/2] [__recompiles] Recompiling function cat in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:222
V0421 01:20:29.253228 47802126279936 torch/_dynamo/guards.py:2611] [34/2] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:29.253228 47802126279936 torch/_dynamo/guards.py:2611] [34/2] [__recompiles]     - tensor 'L['seq'][1]' size mismatch at index 0. expected 2168, actual 564
V0421 01:20:29.253228 47802126279936 torch/_dynamo/guards.py:2611] [34/2] [__recompiles]     - tensor 'L['seq'][0]' size mismatch at index 0. expected 0, actual 4336
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/torch/_dynamo/variables/functions.py:663: UserWarning: Graph break due to unsupported builtin posix.stat. This function is either a Python builtin (e.g. _warnings.warn) or a third-party C/C++ Python extension (perhaps created with pybind). If it is a Python builtin, please file an issue on GitHub so the PyTorch team can add support for it and see the next case for a workaround. If it is a third-party C/C++ Python extension, please either wrap it into a PyTorch-understood custom operator (see https://pytorch.org/tutorials/advanced/custom_ops_landing_page.html for more details) or, if it is traceable, use torch.compiler.allow_in_graph.
  torch._dynamo.utils.warn_once(msg)
V0421 01:20:34.413836 47802126279936 torch/_dynamo/guards.py:2611] [6/3] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:432
V0421 01:20:34.413836 47802126279936 torch/_dynamo/guards.py:2611] [6/3] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:34.413836 47802126279936 torch/_dynamo/guards.py:2611] [6/3] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0421 01:20:34.413836 47802126279936 torch/_dynamo/guards.py:2611] [6/3] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0421 01:20:34.413836 47802126279936 torch/_dynamo/guards.py:2611] [6/3] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0421 01:20:34.460555 47802126279936 torch/_dynamo/guards.py:2611] [7/1] [__recompiles] Recompiling function torch_dynamo_resume_in_zerocopy_to_dgl_ndarray_at_435 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:435
V0421 01:20:34.460555 47802126279936 torch/_dynamo/guards.py:2611] [7/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:34.460555 47802126279936 torch/_dynamo/guards.py:2611] [7/1] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/torch/_dynamo/variables/functions.py:663: UserWarning: Graph break due to unsupported builtin time.perf_counter. This function is either a Python builtin (e.g. _warnings.warn) or a third-party C/C++ Python extension (perhaps created with pybind). If it is a Python builtin, please file an issue on GitHub so the PyTorch team can add support for it and see the next case for a workaround. If it is a third-party C/C++ Python extension, please either wrap it into a PyTorch-understood custom operator (see https://pytorch.org/tutorials/advanced/custom_ops_landing_page.html for more details) or, if it is traceable, use torch.compiler.allow_in_graph.
  torch._dynamo.utils.warn_once(msg)
V0421 01:20:35.001693 47802126279936 torch/_dynamo/guards.py:2611] [86/1] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 01:20:35.001693 47802126279936 torch/_dynamo/guards.py:2611] [86/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:35.001693 47802126279936 torch/_dynamo/guards.py:2611] [86/1] [__recompiles]     - L['___stack0'] == 919876.473741408                          
V0421 01:20:35.095860 47802126279936 torch/_dynamo/guards.py:2611] [86/2] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 01:20:35.095860 47802126279936 torch/_dynamo/guards.py:2611] [86/2] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:35.095860 47802126279936 torch/_dynamo/guards.py:2611] [86/2] [__recompiles]     - L['___stack0'] == 919876.486575691                          
V0421 01:20:35.095860 47802126279936 torch/_dynamo/guards.py:2611] [86/2] [__recompiles]     - L['___stack0'] == 919876.473741408                          
V0421 01:20:38.304577 47802126279936 torch/_dynamo/guards.py:2611] [86/3] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 01:20:38.304577 47802126279936 torch/_dynamo/guards.py:2611] [86/3] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:38.304577 47802126279936 torch/_dynamo/guards.py:2611] [86/3] [__recompiles]     - L['___stack0'] == 919876.580744637                          
V0421 01:20:38.304577 47802126279936 torch/_dynamo/guards.py:2611] [86/3] [__recompiles]     - L['___stack0'] == 919876.486575691                          
V0421 01:20:38.304577 47802126279936 torch/_dynamo/guards.py:2611] [86/3] [__recompiles]     - L['___stack0'] == 919876.473741408                          
V0421 01:20:39.925385 47802126279936 torch/_dynamo/guards.py:2611] [86/4] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 01:20:39.925385 47802126279936 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:39.925385 47802126279936 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     - L['___stack0'] == 919879.789270156                          
V0421 01:20:39.925385 47802126279936 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     - L['___stack0'] == 919876.580744637                          
V0421 01:20:39.925385 47802126279936 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     - L['___stack0'] == 919876.486575691                          
V0421 01:20:39.925385 47802126279936 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     - L['___stack0'] == 919876.473741408                          
V0421 01:20:39.994288 47802126279936 torch/_dynamo/guards.py:2611] [86/5] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 01:20:39.994288 47802126279936 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:39.994288 47802126279936 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 919881.410164426                          
V0421 01:20:39.994288 47802126279936 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 919879.789270156                          
V0421 01:20:39.994288 47802126279936 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 919876.580744637                          
V0421 01:20:39.994288 47802126279936 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 919876.486575691                          
V0421 01:20:39.994288 47802126279936 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 919876.473741408                          
V0421 01:20:40.000522 47802126279936 torch/_dynamo/guards.py:2611] [86/6] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 01:20:40.000522 47802126279936 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:40.000522 47802126279936 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 919881.479127561                          
V0421 01:20:40.000522 47802126279936 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 919881.410164426                          
V0421 01:20:40.000522 47802126279936 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 919879.789270156                          
V0421 01:20:40.000522 47802126279936 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 919876.580744637                          
V0421 01:20:40.000522 47802126279936 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 919876.486575691                          
V0421 01:20:40.000522 47802126279936 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 919876.473741408                          
V0421 01:20:40.065040 47802126279936 torch/_dynamo/guards.py:2611] [57/1] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/view.py:82
V0421 01:20:40.065040 47802126279936 torch/_dynamo/guards.py:2611] [57/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:40.065040 47802126279936 torch/_dynamo/guards.py:2611] [57/1] [__recompiles]     - tensor 'L['val']' requires_grad mismatch. expected requires_grad=0
V0421 01:20:40.087038 47802126279936 torch/_dynamo/guards.py:2611] [61/1] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:690
V0421 01:20:40.087038 47802126279936 torch/_dynamo/guards.py:2611] [61/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:40.087038 47802126279936 torch/_dynamo/guards.py:2611] [61/1] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 01:20:40.106582 47802126279936 torch/_dynamo/guards.py:2611] [62/1] [__recompiles] Recompiling function update_column in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:772
V0421 01:20:40.106582 47802126279936 torch/_dynamo/guards.py:2611] [62/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:40.106582 47802126279936 torch/_dynamo/guards.py:2611] [62/1] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 01:20:40.126265 47802126279936 torch/_dynamo/guards.py:2611] [63/1] [__recompiles] Recompiling function create in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:472
V0421 01:20:40.126265 47802126279936 torch/_dynamo/guards.py:2611] [63/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:40.126265 47802126279936 torch/_dynamo/guards.py:2611] [63/1] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 01:20:40.143768 47802126279936 torch/_dynamo/guards.py:2611] [64/1] [__recompiles] Recompiling function __init__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:210
V0421 01:20:40.143768 47802126279936 torch/_dynamo/guards.py:2611] [64/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:40.143768 47802126279936 torch/_dynamo/guards.py:2611] [64/1] [__recompiles]     - tensor 'L['storage']' requires_grad mismatch. expected requires_grad=0
V0421 01:20:40.172616 47802126279936 torch/_dynamo/guards.py:2611] [65/1] [__recompiles] Recompiling function infer_scheme in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:151
V0421 01:20:40.172616 47802126279936 torch/_dynamo/guards.py:2611] [65/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:40.172616 47802126279936 torch/_dynamo/guards.py:2611] [65/1] [__recompiles]     - tensor 'L['tensor']' requires_grad mismatch. expected requires_grad=0
V0421 01:20:40.220925 47802126279936 torch/_dynamo/guards.py:2611] [68/1] [__recompiles] Recompiling function func in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:213
V0421 01:20:40.220925 47802126279936 torch/_dynamo/guards.py:2611] [68/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:40.220925 47802126279936 torch/_dynamo/guards.py:2611] [68/1] [__recompiles]     - tensor 'L['x']' requires_grad mismatch. expected requires_grad=0
V0421 01:20:40.234620 47802126279936 torch/_dynamo/guards.py:2611] [70/1] [__recompiles] Recompiling function torch_dynamo_resume_in_gspmm_at_75 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:75
V0421 01:20:40.234620 47802126279936 torch/_dynamo/guards.py:2611] [70/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:40.234620 47802126279936 torch/_dynamo/guards.py:2611] [70/1] [__recompiles]     - tensor 'L['lhs_data']' requires_grad mismatch. expected requires_grad=0
V0421 01:20:40.252532 47802126279936 torch/_dynamo/guards.py:2611] [74/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/sparse.py:163
V0421 01:20:40.252532 47802126279936 torch/_dynamo/guards.py:2611] [74/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:40.252532 47802126279936 torch/_dynamo/guards.py:2611] [74/1] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=0
V0421 01:20:40.268717 47802126279936 torch/_dynamo/guards.py:2611] [77/1] [__recompiles] Recompiling function zeros in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:287
V0421 01:20:40.268717 47802126279936 torch/_dynamo/guards.py:2611] [77/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:40.268717 47802126279936 torch/_dynamo/guards.py:2611] [77/1] [__recompiles]     - L['shape'][1] == 1433                                       
V0421 01:20:40.531365 47802126279936 torch/_dynamo/guards.py:2611] [6/4] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:432
V0421 01:20:40.531365 47802126279936 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:40.531365 47802126279936 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0421 01:20:40.531365 47802126279936 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 01:20:40.531365 47802126279936 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0421 01:20:40.531365 47802126279936 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0421 01:20:40.581289 47802126279936 torch/_dynamo/guards.py:2611] [79/1] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray_for_write in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:454
V0421 01:20:40.581289 47802126279936 torch/_dynamo/guards.py:2611] [79/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:40.581289 47802126279936 torch/_dynamo/guards.py:2611] [79/1] [__recompiles]     - tensor 'L['input']' stride mismatch at index 0. expected 1433, actual 128
V0421 01:20:40.618077 47802126279936 torch/_dynamo/guards.py:2611] [80/1] [__recompiles] Recompiling function check_is_view in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:441
V0421 01:20:40.618077 47802126279936 torch/_dynamo/guards.py:2611] [80/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:40.618077 47802126279936 torch/_dynamo/guards.py:2611] [80/1] [__recompiles]     - tensor 'L['input']' stride mismatch at index 0. expected 1433, actual 128
V0421 01:20:40.628103 47802126279936 torch/_dynamo/guards.py:2611] [81/1] [__recompiles] Recompiling function torch_dynamo_resume_in_check_is_view_at_443 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:443
V0421 01:20:40.628103 47802126279936 torch/_dynamo/guards.py:2611] [81/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:40.628103 47802126279936 torch/_dynamo/guards.py:2611] [81/1] [__recompiles]     - tensor 'L['input']' stride mismatch at index 0. expected 1433, actual 128
V0421 01:20:40.639513 47802126279936 torch/_dynamo/guards.py:2611] [82/1] [__recompiles] Recompiling function torch_dynamo_resume_in_zerocopy_to_dgl_ndarray_for_write_at_461 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:461
V0421 01:20:40.639513 47802126279936 torch/_dynamo/guards.py:2611] [82/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:40.639513 47802126279936 torch/_dynamo/guards.py:2611] [82/1] [__recompiles]     - tensor 'L['input']' stride mismatch at index 0. expected 1433, actual 128
V0421 01:20:40.652983 47802126279936 torch/_dynamo/guards.py:2611] [83/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_165 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/sparse.py:165
V0421 01:20:40.652983 47802126279936 torch/_dynamo/guards.py:2611] [83/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:40.652983 47802126279936 torch/_dynamo/guards.py:2611] [83/1] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=0
V0421 01:20:40.692346 47802126279936 torch/_dynamo/guards.py:2611] [86/7] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 01:20:40.692346 47802126279936 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:40.692346 47802126279936 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 919881.485361779                          
V0421 01:20:40.692346 47802126279936 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 919881.479127561                          
V0421 01:20:40.692346 47802126279936 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 919881.410164426                          
V0421 01:20:40.692346 47802126279936 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 919879.789270156                          
V0421 01:20:40.692346 47802126279936 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 919876.580744637                          
V0421 01:20:40.692346 47802126279936 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 919876.486575691                          
V0421 01:20:40.692346 47802126279936 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 919876.473741408                          
V0421 01:20:40.699121 47802126279936 torch/_dynamo/guards.py:2611] [86/8] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 01:20:40.699121 47802126279936 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:40.699121 47802126279936 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 919882.177127219                          
V0421 01:20:40.699121 47802126279936 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 919881.485361779                          
V0421 01:20:40.699121 47802126279936 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 919881.479127561                          
V0421 01:20:40.699121 47802126279936 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 919881.410164426                          
V0421 01:20:40.699121 47802126279936 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 919879.789270156                          
V0421 01:20:40.699121 47802126279936 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 919876.580744637                          
V0421 01:20:40.699121 47802126279936 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 919876.486575691                          
V0421 01:20:40.699121 47802126279936 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 919876.473741408                          
W0421 01:20:40.699261 47802126279936 torch/_dynamo/convert_frame.py:762] [86/8] torch._dynamo hit config.cache_size_limit (8)
W0421 01:20:40.699261 47802126279936 torch/_dynamo/convert_frame.py:762] [86/8]    function: 'torch_dynamo_resume_in_hook_at_12' (/work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12)
W0421 01:20:40.699261 47802126279936 torch/_dynamo/convert_frame.py:762] [86/8]    last reason: L['___stack0'] == 919876.473741408                          
W0421 01:20:40.699261 47802126279936 torch/_dynamo/convert_frame.py:762] [86/8] To log all recompilation reasons, use TORCH_LOGS="recompiles".
W0421 01:20:40.699261 47802126279936 torch/_dynamo/convert_frame.py:762] [86/8] To diagnose recompilation issues, see https://pytorch.org/docs/main/torch.compiler_troubleshooting.html.
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 01:20:41.604458 47802126279936 torch/_dynamo/guards.py:2611] [0/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/VQGraph/models.py:449
V0421 01:20:41.604458 47802126279936 torch/_dynamo/guards.py:2611] [0/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:41.604458 47802126279936 torch/_dynamo/guards.py:2611] [0/1] [__recompiles]     - tensor 'L['feats']' size mismatch at index 0. expected 1049, actual 1048
V0421 01:20:41.629856 47802126279936 torch/_dynamo/guards.py:2611] [1/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/VQGraph/models.py:154
V0421 01:20:41.629856 47802126279936 torch/_dynamo/guards.py:2611] [1/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:41.629856 47802126279936 torch/_dynamo/guards.py:2611] [1/1] [__recompiles]     - tensor 'L['feats']' size mismatch at index 0. expected 1049, actual 1048
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 01:20:41.646017 47802126279936 torch/_dynamo/guards.py:2611] [9/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_157 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/models.py:157
V0421 01:20:41.646017 47802126279936 torch/_dynamo/guards.py:2611] [9/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:41.646017 47802126279936 torch/_dynamo/guards.py:2611] [9/1] [__recompiles]     - tensor 'L['h']' size mismatch at index 0. expected 1049, actual 1048
V0421 01:20:41.660807 47802126279936 torch/_dynamo/guards.py:2611] [13/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_157 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/models.py:157
V0421 01:20:41.660807 47802126279936 torch/_dynamo/guards.py:2611] [13/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:41.660807 47802126279936 torch/_dynamo/guards.py:2611] [13/1] [__recompiles]     - tensor 'L['h']' size mismatch at index 0. expected 1049, actual 1048
V0421 01:20:41.701914 47802126279936 torch/_dynamo/guards.py:2611] [4/3] [__recompiles] Recompiling function create_unitgraph_from_coo in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1254
V0421 01:20:41.701914 47802126279936 torch/_dynamo/guards.py:2611] [4/3] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:41.701914 47802126279936 torch/_dynamo/guards.py:2611] [4/3] [__recompiles]     - tensor 'L['row']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 01:20:41.701914 47802126279936 torch/_dynamo/guards.py:2611] [4/3] [__recompiles]     - L['num_dst'] == 1049                                        
V0421 01:20:41.701914 47802126279936 torch/_dynamo/guards.py:2611] [4/3] [__recompiles]     - L['num_dst'] == 1049                                        
V0421 01:20:41.733931 47802126279936 torch/_dynamo/guards.py:2611] [8/3] [__recompiles] Recompiling function torch_dynamo_resume_in_create_unitgraph_from_coo_at_1296 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1296
V0421 01:20:41.733931 47802126279936 torch/_dynamo/guards.py:2611] [8/3] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:41.733931 47802126279936 torch/_dynamo/guards.py:2611] [8/3] [__recompiles]     - tensor 'L['col']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 01:20:41.733931 47802126279936 torch/_dynamo/guards.py:2611] [8/3] [__recompiles]     - L['___stack2'] == 1049                                      
V0421 01:20:41.733931 47802126279936 torch/_dynamo/guards.py:2611] [8/3] [__recompiles]     - L['___stack2'] == 1049                                      
V0421 01:20:41.747913 47802126279936 torch/_dynamo/guards.py:2611] [15/1] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 01:20:41.747913 47802126279936 torch/_dynamo/guards.py:2611] [15/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:41.747913 47802126279936 torch/_dynamo/guards.py:2611] [15/1] [__recompiles]     - L['data'][0] == 1049                                        
V0421 01:20:44.963150 47802126279936 torch/_dynamo/guards.py:2611] [29/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_572 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:572
V0421 01:20:44.963150 47802126279936 torch/_dynamo/guards.py:2611] [29/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:44.963150 47802126279936 torch/_dynamo/guards.py:2611] [29/1] [__recompiles]     - L['___stack0'] == 1049                                      
V0421 01:20:44.980881 47802126279936 torch/_dynamo/guards.py:2611] [30/2] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_573 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:573
V0421 01:20:44.980881 47802126279936 torch/_dynamo/guards.py:2611] [30/2] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:44.980881 47802126279936 torch/_dynamo/guards.py:2611] [30/2] [__recompiles]     - L['___stack0'] == 1049                                      
V0421 01:20:44.980881 47802126279936 torch/_dynamo/guards.py:2611] [30/2] [__recompiles]     - tensor 'L['u']' size mismatch at index 0. expected 2168, actual 2179
V0421 01:20:45.439187 47802126279936 torch/_dynamo/guards.py:2611] [34/3] [__recompiles] Recompiling function cat in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:222
V0421 01:20:45.439187 47802126279936 torch/_dynamo/guards.py:2611] [34/3] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:45.439187 47802126279936 torch/_dynamo/guards.py:2611] [34/3] [__recompiles]     - 2 <= L['seq'][0].size()[0] <= 2147483645                      # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 01:20:45.439187 47802126279936 torch/_dynamo/guards.py:2611] [34/3] [__recompiles]     - tensor 'L['seq'][1]' size mismatch at index 0. expected 2168, actual 2179
V0421 01:20:45.439187 47802126279936 torch/_dynamo/guards.py:2611] [34/3] [__recompiles]     - tensor 'L['seq'][1]' size mismatch at index 0. expected 2168, actual 2179
V0421 01:20:46.067071 47802126279936 torch/_dynamo/guards.py:2611] [4/4] [__recompiles] Recompiling function create_unitgraph_from_coo in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1254
V0421 01:20:46.067071 47802126279936 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:46.067071 47802126279936 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     - tensor 'L['row']' size mismatch at index 0. expected 0, actual 2179
V0421 01:20:46.067071 47802126279936 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     - tensor 'L['row']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 01:20:46.067071 47802126279936 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     - L['num_dst'] == 1049                                        
V0421 01:20:46.067071 47802126279936 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     - tensor 'L['row']' size mismatch at index 0. expected 0, actual 2179
V0421 01:20:46.106647 47802126279936 torch/_dynamo/guards.py:2611] [8/4] [__recompiles] Recompiling function torch_dynamo_resume_in_create_unitgraph_from_coo_at_1296 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1296
V0421 01:20:46.106647 47802126279936 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:46.106647 47802126279936 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     - tensor 'L['col']' size mismatch at index 0. expected 0, actual 2179
V0421 01:20:46.106647 47802126279936 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     - tensor 'L['col']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 01:20:46.106647 47802126279936 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     - L['___stack2'] == 1049                                      
V0421 01:20:46.106647 47802126279936 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     - tensor 'L['col']' size mismatch at index 0. expected 0, actual 2179
V0421 01:20:46.127127 47802126279936 torch/_dynamo/guards.py:2611] [41/1] [__recompiles] Recompiling function spmatrix in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/sparse/sparse_matrix.py:763
V0421 01:20:46.127127 47802126279936 torch/_dynamo/guards.py:2611] [41/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:46.127127 47802126279936 torch/_dynamo/guards.py:2611] [41/1] [__recompiles]     - tensor 'L['indices']' stride mismatch at index 0. expected 5464, actual 5486
V0421 01:20:46.766127 47802126279936 torch/_dynamo/guards.py:2611] [47/1] [__recompiles] Recompiling function arange in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:406
V0421 01:20:46.766127 47802126279936 torch/_dynamo/guards.py:2611] [47/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:46.766127 47802126279936 torch/_dynamo/guards.py:2611] [47/1] [__recompiles]     - L['stop'] == 1049                                           
V0421 01:20:47.670896 47802126279936 torch/_dynamo/guards.py:2611] [48/1] [__recompiles] Recompiling function in_degrees in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:702
V0421 01:20:47.670896 47802126279936 torch/_dynamo/guards.py:2611] [48/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:47.670896 47802126279936 torch/_dynamo/guards.py:2611] [48/1] [__recompiles]     - tensor 'L['v']' size mismatch at index 0. expected 1049, actual 1048
V0421 01:20:47.688699 47802126279936 torch/_dynamo/guards.py:2611] [50/1] [__recompiles] Recompiling function has_nodes in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:2878
V0421 01:20:47.688699 47802126279936 torch/_dynamo/guards.py:2611] [50/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:47.688699 47802126279936 torch/_dynamo/guards.py:2611] [50/1] [__recompiles]     - tensor 'L['vid']' size mismatch at index 0. expected 1049, actual 1048
V0421 01:20:47.704786 47802126279936 torch/_dynamo/guards.py:2611] [51/1] [__recompiles] Recompiling function torch_dynamo_resume_in_has_nodes_at_2925 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:2925
V0421 01:20:47.704786 47802126279936 torch/_dynamo/guards.py:2611] [51/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:47.704786 47802126279936 torch/_dynamo/guards.py:2611] [51/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 1049, actual 1048
V0421 01:20:48.150951 47802126279936 torch/_dynamo/guards.py:2611] [52/1] [__recompiles] Recompiling function torch_dynamo_resume_in_has_nodes_at_2926 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:2926
V0421 01:20:48.150951 47802126279936 torch/_dynamo/guards.py:2611] [52/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:48.150951 47802126279936 torch/_dynamo/guards.py:2611] [52/1] [__recompiles]     - tensor 'L['vid_tensor']' size mismatch at index 0. expected 1049, actual 1048
V0421 01:20:48.175567 47802126279936 torch/_dynamo/guards.py:2611] [53/1] [__recompiles] Recompiling function has_nodes in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:444
V0421 01:20:48.175567 47802126279936 torch/_dynamo/guards.py:2611] [53/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:48.175567 47802126279936 torch/_dynamo/guards.py:2611] [53/1] [__recompiles]     - tensor 'L['vids']' size mismatch at index 0. expected 1049, actual 1048
V0421 01:20:48.192394 47802126279936 torch/_dynamo/guards.py:2611] [54/1] [__recompiles] Recompiling function torch_dynamo_resume_in_has_nodes_at_2930 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:2930
V0421 01:20:48.192394 47802126279936 torch/_dynamo/guards.py:2611] [54/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:48.192394 47802126279936 torch/_dynamo/guards.py:2611] [54/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 1049, actual 1048
V0421 01:20:48.793693 47802126279936 torch/_dynamo/guards.py:2611] [55/1] [__recompiles] Recompiling function sum in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:152
V0421 01:20:48.793693 47802126279936 torch/_dynamo/guards.py:2611] [55/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:48.793693 47802126279936 torch/_dynamo/guards.py:2611] [55/1] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 1049, actual 1048
V0421 01:20:49.069003 47802126279936 torch/_dynamo/guards.py:2611] [57/2] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/view.py:82
V0421 01:20:49.069003 47802126279936 torch/_dynamo/guards.py:2611] [57/2] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:49.069003 47802126279936 torch/_dynamo/guards.py:2611] [57/2] [__recompiles]     - tensor 'L['val']' requires_grad mismatch. expected requires_grad=1
V0421 01:20:49.069003 47802126279936 torch/_dynamo/guards.py:2611] [57/2] [__recompiles]     - tensor 'L['val']' size mismatch at index 0. expected 1049, actual 1048
V0421 01:20:49.087459 47802126279936 torch/_dynamo/guards.py:2611] [61/2] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:690
V0421 01:20:49.087459 47802126279936 torch/_dynamo/guards.py:2611] [61/2] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:49.087459 47802126279936 torch/_dynamo/guards.py:2611] [61/2] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=1
V0421 01:20:49.087459 47802126279936 torch/_dynamo/guards.py:2611] [61/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 1049, actual 1048
V0421 01:20:49.105783 47802126279936 torch/_dynamo/guards.py:2611] [62/2] [__recompiles] Recompiling function update_column in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:772
V0421 01:20:49.105783 47802126279936 torch/_dynamo/guards.py:2611] [62/2] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:49.105783 47802126279936 torch/_dynamo/guards.py:2611] [62/2] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=1
V0421 01:20:49.105783 47802126279936 torch/_dynamo/guards.py:2611] [62/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 1049, actual 1048
V0421 01:20:49.124693 47802126279936 torch/_dynamo/guards.py:2611] [63/2] [__recompiles] Recompiling function create in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:472
V0421 01:20:49.124693 47802126279936 torch/_dynamo/guards.py:2611] [63/2] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:49.124693 47802126279936 torch/_dynamo/guards.py:2611] [63/2] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=1
V0421 01:20:49.124693 47802126279936 torch/_dynamo/guards.py:2611] [63/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 1049, actual 1048
V0421 01:20:49.141365 47802126279936 torch/_dynamo/guards.py:2611] [64/2] [__recompiles] Recompiling function __init__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:210
V0421 01:20:49.141365 47802126279936 torch/_dynamo/guards.py:2611] [64/2] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:49.141365 47802126279936 torch/_dynamo/guards.py:2611] [64/2] [__recompiles]     - tensor 'L['storage']' requires_grad mismatch. expected requires_grad=1
V0421 01:20:49.141365 47802126279936 torch/_dynamo/guards.py:2611] [64/2] [__recompiles]     - tensor 'L['storage']' size mismatch at index 0. expected 1049, actual 1048
V0421 01:20:49.160066 47802126279936 torch/_dynamo/guards.py:2611] [65/2] [__recompiles] Recompiling function infer_scheme in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:151
V0421 01:20:49.160066 47802126279936 torch/_dynamo/guards.py:2611] [65/2] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:49.160066 47802126279936 torch/_dynamo/guards.py:2611] [65/2] [__recompiles]     - tensor 'L['tensor']' requires_grad mismatch. expected requires_grad=1
V0421 01:20:49.160066 47802126279936 torch/_dynamo/guards.py:2611] [65/2] [__recompiles]     - tensor 'L['tensor']' size mismatch at index 0. expected 1049, actual 1048
V0421 01:20:49.203944 47802126279936 torch/_dynamo/guards.py:2611] [68/2] [__recompiles] Recompiling function func in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:213
V0421 01:20:49.203944 47802126279936 torch/_dynamo/guards.py:2611] [68/2] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:49.203944 47802126279936 torch/_dynamo/guards.py:2611] [68/2] [__recompiles]     - tensor 'L['x']' requires_grad mismatch. expected requires_grad=1
V0421 01:20:49.203944 47802126279936 torch/_dynamo/guards.py:2611] [68/2] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 1049, actual 1048
V0421 01:20:49.217395 47802126279936 torch/_dynamo/guards.py:2611] [70/2] [__recompiles] Recompiling function torch_dynamo_resume_in_gspmm_at_75 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:75
V0421 01:20:49.217395 47802126279936 torch/_dynamo/guards.py:2611] [70/2] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:49.217395 47802126279936 torch/_dynamo/guards.py:2611] [70/2] [__recompiles]     - tensor 'L['lhs_data']' requires_grad mismatch. expected requires_grad=1
V0421 01:20:49.217395 47802126279936 torch/_dynamo/guards.py:2611] [70/2] [__recompiles]     - tensor 'L['lhs_data']' size mismatch at index 0. expected 1049, actual 1048
V0421 01:20:49.235855 47802126279936 torch/_dynamo/guards.py:2611] [74/2] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/sparse.py:163
V0421 01:20:49.235855 47802126279936 torch/_dynamo/guards.py:2611] [74/2] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:49.235855 47802126279936 torch/_dynamo/guards.py:2611] [74/2] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=1
V0421 01:20:49.235855 47802126279936 torch/_dynamo/guards.py:2611] [74/2] [__recompiles]     - tensor 'L['X']' size mismatch at index 0. expected 1049, actual 1048
V0421 01:20:49.251965 47802126279936 torch/_dynamo/guards.py:2611] [77/2] [__recompiles] Recompiling function zeros in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:287
V0421 01:20:49.251965 47802126279936 torch/_dynamo/guards.py:2611] [77/2] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:49.251965 47802126279936 torch/_dynamo/guards.py:2611] [77/2] [__recompiles]     - L['shape'][0] == 1049                                       
V0421 01:20:49.251965 47802126279936 torch/_dynamo/guards.py:2611] [77/2] [__recompiles]     - L['shape'][0] == 1049                                       
V0421 01:20:49.505959 47802126279936 torch/_dynamo/guards.py:2611] [79/2] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray_for_write in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:454
V0421 01:20:49.505959 47802126279936 torch/_dynamo/guards.py:2611] [79/2] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:49.505959 47802126279936 torch/_dynamo/guards.py:2611] [79/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 1049, actual 1048
V0421 01:20:49.505959 47802126279936 torch/_dynamo/guards.py:2611] [79/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 1049, actual 1048
V0421 01:20:49.548308 47802126279936 torch/_dynamo/guards.py:2611] [80/2] [__recompiles] Recompiling function check_is_view in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:441
V0421 01:20:49.548308 47802126279936 torch/_dynamo/guards.py:2611] [80/2] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:49.548308 47802126279936 torch/_dynamo/guards.py:2611] [80/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 1049, actual 1048
V0421 01:20:49.548308 47802126279936 torch/_dynamo/guards.py:2611] [80/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 1049, actual 1048
V0421 01:20:49.559832 47802126279936 torch/_dynamo/guards.py:2611] [81/2] [__recompiles] Recompiling function torch_dynamo_resume_in_check_is_view_at_443 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:443
V0421 01:20:49.559832 47802126279936 torch/_dynamo/guards.py:2611] [81/2] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:49.559832 47802126279936 torch/_dynamo/guards.py:2611] [81/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 1049, actual 1048
V0421 01:20:49.559832 47802126279936 torch/_dynamo/guards.py:2611] [81/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 1049, actual 1048
V0421 01:20:49.572599 47802126279936 torch/_dynamo/guards.py:2611] [82/2] [__recompiles] Recompiling function torch_dynamo_resume_in_zerocopy_to_dgl_ndarray_for_write_at_461 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:461
V0421 01:20:49.572599 47802126279936 torch/_dynamo/guards.py:2611] [82/2] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:49.572599 47802126279936 torch/_dynamo/guards.py:2611] [82/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 1049, actual 1048
V0421 01:20:49.572599 47802126279936 torch/_dynamo/guards.py:2611] [82/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 1049, actual 1048
V0421 01:20:49.588963 47802126279936 torch/_dynamo/guards.py:2611] [88/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:569
V0421 01:20:49.588963 47802126279936 torch/_dynamo/guards.py:2611] [88/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:49.588963 47802126279936 torch/_dynamo/guards.py:2611] [88/1] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 1049, actual 1048
V0421 01:20:49.679983 47802126279936 torch/_dynamo/guards.py:2611] [89/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_590 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:590
V0421 01:20:49.679983 47802126279936 torch/_dynamo/guards.py:2611] [89/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:49.679983 47802126279936 torch/_dynamo/guards.py:2611] [89/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 1049, actual 1048
V0421 01:20:49.740559 47802126279936 torch/_dynamo/guards.py:2611] [90/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:418
V0421 01:20:49.740559 47802126279936 torch/_dynamo/guards.py:2611] [90/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:49.740559 47802126279936 torch/_dynamo/guards.py:2611] [90/1] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 1049, actual 1048
V0421 01:20:50.762416 47802126279936 torch/_dynamo/guards.py:2611] [93/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_432 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:432
V0421 01:20:50.762416 47802126279936 torch/_dynamo/guards.py:2611] [93/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:50.762416 47802126279936 torch/_dynamo/guards.py:2611] [93/1] [__recompiles]     - L['shape'][1] == 1049                                       
V0421 01:20:53.353687 47802126279936 torch/_dynamo/guards.py:2611] [94/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_595 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:595
V0421 01:20:53.353687 47802126279936 torch/_dynamo/guards.py:2611] [94/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:53.353687 47802126279936 torch/_dynamo/guards.py:2611] [94/1] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 1049, actual 1048
V0421 01:20:53.996471 47802126279936 torch/_dynamo/guards.py:2611] [95/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_604 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:604
V0421 01:20:53.996471 47802126279936 torch/_dynamo/guards.py:2611] [95/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:53.996471 47802126279936 torch/_dynamo/guards.py:2611] [95/1] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 1049, actual 1048
V0421 01:20:55.695985 47802126279936 torch/_dynamo/guards.py:2611] [96/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_648 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:648
V0421 01:20:55.695985 47802126279936 torch/_dynamo/guards.py:2611] [96/1] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:55.695985 47802126279936 torch/_dynamo/guards.py:2611] [96/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 1049, actual 1048
V0421 01:20:55.800595 47802126279936 torch/_dynamo/guards.py:2611] [57/3] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/view.py:82
V0421 01:20:55.800595 47802126279936 torch/_dynamo/guards.py:2611] [57/3] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:55.800595 47802126279936 torch/_dynamo/guards.py:2611] [57/3] [__recompiles]     - tensor 'L['val']' requires_grad mismatch. expected requires_grad=0
V0421 01:20:55.800595 47802126279936 torch/_dynamo/guards.py:2611] [57/3] [__recompiles]     - tensor 'L['val']' size mismatch at index 0. expected 1049, actual 1048
V0421 01:20:55.800595 47802126279936 torch/_dynamo/guards.py:2611] [57/3] [__recompiles]     - tensor 'L['val']' requires_grad mismatch. expected requires_grad=0
V0421 01:20:55.822615 47802126279936 torch/_dynamo/guards.py:2611] [61/3] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:690
V0421 01:20:55.822615 47802126279936 torch/_dynamo/guards.py:2611] [61/3] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:55.822615 47802126279936 torch/_dynamo/guards.py:2611] [61/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 01:20:55.822615 47802126279936 torch/_dynamo/guards.py:2611] [61/3] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 1049, actual 1048
V0421 01:20:55.822615 47802126279936 torch/_dynamo/guards.py:2611] [61/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 01:20:55.843614 47802126279936 torch/_dynamo/guards.py:2611] [62/3] [__recompiles] Recompiling function update_column in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:772
V0421 01:20:55.843614 47802126279936 torch/_dynamo/guards.py:2611] [62/3] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:55.843614 47802126279936 torch/_dynamo/guards.py:2611] [62/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 01:20:55.843614 47802126279936 torch/_dynamo/guards.py:2611] [62/3] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 1049, actual 1048
V0421 01:20:55.843614 47802126279936 torch/_dynamo/guards.py:2611] [62/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 01:20:55.864803 47802126279936 torch/_dynamo/guards.py:2611] [63/3] [__recompiles] Recompiling function create in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:472
V0421 01:20:55.864803 47802126279936 torch/_dynamo/guards.py:2611] [63/3] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:55.864803 47802126279936 torch/_dynamo/guards.py:2611] [63/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 01:20:55.864803 47802126279936 torch/_dynamo/guards.py:2611] [63/3] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 1049, actual 1048
V0421 01:20:55.864803 47802126279936 torch/_dynamo/guards.py:2611] [63/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 01:20:55.883771 47802126279936 torch/_dynamo/guards.py:2611] [64/3] [__recompiles] Recompiling function __init__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:210
V0421 01:20:55.883771 47802126279936 torch/_dynamo/guards.py:2611] [64/3] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:55.883771 47802126279936 torch/_dynamo/guards.py:2611] [64/3] [__recompiles]     - tensor 'L['storage']' requires_grad mismatch. expected requires_grad=0
V0421 01:20:55.883771 47802126279936 torch/_dynamo/guards.py:2611] [64/3] [__recompiles]     - tensor 'L['storage']' size mismatch at index 0. expected 1049, actual 1048
V0421 01:20:55.883771 47802126279936 torch/_dynamo/guards.py:2611] [64/3] [__recompiles]     - tensor 'L['storage']' requires_grad mismatch. expected requires_grad=0
V0421 01:20:55.904848 47802126279936 torch/_dynamo/guards.py:2611] [65/3] [__recompiles] Recompiling function infer_scheme in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:151
V0421 01:20:55.904848 47802126279936 torch/_dynamo/guards.py:2611] [65/3] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:55.904848 47802126279936 torch/_dynamo/guards.py:2611] [65/3] [__recompiles]     - tensor 'L['tensor']' requires_grad mismatch. expected requires_grad=0
V0421 01:20:55.904848 47802126279936 torch/_dynamo/guards.py:2611] [65/3] [__recompiles]     - tensor 'L['tensor']' size mismatch at index 0. expected 1049, actual 1048
V0421 01:20:55.904848 47802126279936 torch/_dynamo/guards.py:2611] [65/3] [__recompiles]     - tensor 'L['tensor']' requires_grad mismatch. expected requires_grad=0
V0421 01:20:55.952901 47802126279936 torch/_dynamo/guards.py:2611] [68/3] [__recompiles] Recompiling function func in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:213
V0421 01:20:55.952901 47802126279936 torch/_dynamo/guards.py:2611] [68/3] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:55.952901 47802126279936 torch/_dynamo/guards.py:2611] [68/3] [__recompiles]     - tensor 'L['x']' requires_grad mismatch. expected requires_grad=0
V0421 01:20:55.952901 47802126279936 torch/_dynamo/guards.py:2611] [68/3] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 1049, actual 1048
V0421 01:20:55.952901 47802126279936 torch/_dynamo/guards.py:2611] [68/3] [__recompiles]     - tensor 'L['x']' requires_grad mismatch. expected requires_grad=0
V0421 01:20:55.968800 47802126279936 torch/_dynamo/guards.py:2611] [70/3] [__recompiles] Recompiling function torch_dynamo_resume_in_gspmm_at_75 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:75
V0421 01:20:55.968800 47802126279936 torch/_dynamo/guards.py:2611] [70/3] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:55.968800 47802126279936 torch/_dynamo/guards.py:2611] [70/3] [__recompiles]     - tensor 'L['lhs_data']' requires_grad mismatch. expected requires_grad=0
V0421 01:20:55.968800 47802126279936 torch/_dynamo/guards.py:2611] [70/3] [__recompiles]     - tensor 'L['lhs_data']' size mismatch at index 0. expected 1049, actual 1048
V0421 01:20:55.968800 47802126279936 torch/_dynamo/guards.py:2611] [70/3] [__recompiles]     - tensor 'L['lhs_data']' requires_grad mismatch. expected requires_grad=0
V0421 01:20:55.988361 47802126279936 torch/_dynamo/guards.py:2611] [74/3] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/sparse.py:163
V0421 01:20:55.988361 47802126279936 torch/_dynamo/guards.py:2611] [74/3] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:55.988361 47802126279936 torch/_dynamo/guards.py:2611] [74/3] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=0
V0421 01:20:55.988361 47802126279936 torch/_dynamo/guards.py:2611] [74/3] [__recompiles]     - tensor 'L['X']' size mismatch at index 0. expected 1049, actual 1048
V0421 01:20:55.988361 47802126279936 torch/_dynamo/guards.py:2611] [74/3] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=0
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 01:20:56.184168 47802126279936 torch/_dynamo/guards.py:2611] [15/2] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 01:20:56.184168 47802126279936 torch/_dynamo/guards.py:2611] [15/2] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:56.184168 47802126279936 torch/_dynamo/guards.py:2611] [15/2] [__recompiles]     - L['data'][0] == 1048                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 01:20:56.184168 47802126279936 torch/_dynamo/guards.py:2611] [15/2] [__recompiles]     - L['data'][0] == 1049                                        
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 01:20:59.533304 47802126279936 torch/_dynamo/guards.py:2611] [15/3] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 01:20:59.533304 47802126279936 torch/_dynamo/guards.py:2611] [15/3] [__recompiles]     triggered by the following guard failure(s):
V0421 01:20:59.533304 47802126279936 torch/_dynamo/guards.py:2611] [15/3] [__recompiles]     - L['data'][0] == 1039                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 01:20:59.533304 47802126279936 torch/_dynamo/guards.py:2611] [15/3] [__recompiles]     - L['data'][0] == 1048                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 01:20:59.533304 47802126279936 torch/_dynamo/guards.py:2611] [15/3] [__recompiles]     - L['data'][0] == 1049                                        
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 01:21:02.878890 47802126279936 torch/_dynamo/guards.py:2611] [15/4] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 01:21:02.878890 47802126279936 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     triggered by the following guard failure(s):
V0421 01:21:02.878890 47802126279936 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     - L['data'][0] == 1060                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 01:21:02.878890 47802126279936 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     - L['data'][0] == 1039                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 01:21:02.878890 47802126279936 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     - L['data'][0] == 1048                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 01:21:02.878890 47802126279936 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     - L['data'][0] == 1049                                        
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 01:21:06.257676 47802126279936 torch/_dynamo/guards.py:2611] [15/5] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 01:21:06.257676 47802126279936 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     triggered by the following guard failure(s):
V0421 01:21:06.257676 47802126279936 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 1028                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 01:21:06.257676 47802126279936 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 1060                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 01:21:06.257676 47802126279936 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 1039                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 01:21:06.257676 47802126279936 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 1048                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 01:21:06.257676 47802126279936 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 1049                                        
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 01:21:09.561652 47802126279936 torch/_dynamo/guards.py:2611] [15/6] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 01:21:09.561652 47802126279936 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     triggered by the following guard failure(s):
V0421 01:21:09.561652 47802126279936 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 1084                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 01:21:09.561652 47802126279936 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 1028                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 01:21:09.561652 47802126279936 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 1060                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 01:21:09.561652 47802126279936 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 1039                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 01:21:09.561652 47802126279936 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 1048                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 01:21:09.561652 47802126279936 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 1049                                        
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 01:21:12.910624 47802126279936 torch/_dynamo/guards.py:2611] [15/7] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 01:21:12.910624 47802126279936 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     triggered by the following guard failure(s):
V0421 01:21:12.910624 47802126279936 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 1052                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 01:21:12.910624 47802126279936 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 1084                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 01:21:12.910624 47802126279936 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 1028                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 01:21:12.910624 47802126279936 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 1060                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 01:21:12.910624 47802126279936 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 1039                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 01:21:12.910624 47802126279936 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 1048                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 01:21:12.910624 47802126279936 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 1049                                        
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 01:21:16.255718 47802126279936 torch/_dynamo/guards.py:2611] [15/8] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 01:21:16.255718 47802126279936 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     triggered by the following guard failure(s):
V0421 01:21:16.255718 47802126279936 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 1040                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 01:21:16.255718 47802126279936 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 1052                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 01:21:16.255718 47802126279936 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 1084                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 01:21:16.255718 47802126279936 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 1028                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 01:21:16.255718 47802126279936 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 1060                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 01:21:16.255718 47802126279936 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 1039                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 01:21:16.255718 47802126279936 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 1048                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 01:21:16.255718 47802126279936 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 1049                                        
W0421 01:21:16.255917 47802126279936 torch/_dynamo/convert_frame.py:762] [15/8] torch._dynamo hit config.cache_size_limit (8)
W0421 01:21:16.255917 47802126279936 torch/_dynamo/convert_frame.py:762] [15/8]    function: '_dispatch' (/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65)
W0421 01:21:16.255917 47802126279936 torch/_dynamo/convert_frame.py:762] [15/8]    last reason: L['data'][0] == 1049                                        
W0421 01:21:16.255917 47802126279936 torch/_dynamo/convert_frame.py:762] [15/8] To log all recompilation reasons, use TORCH_LOGS="recompiles".
W0421 01:21:16.255917 47802126279936 torch/_dynamo/convert_frame.py:762] [15/8] To diagnose recompilation issues, see https://pytorch.org/docs/main/torch.compiler_troubleshooting.html.
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
Compiling model with torch.compile()...
Saved sparsity snapshot to outputs_compiled/tran/cora/SAGE/seed_0/sparsity_before.csv
                               layer_name  num_elements    sparsity
0  _orig_mod.encoder.graph_layer_1.weight       2053489    0.000000
1    _orig_mod.encoder.graph_layer_1.bias          1433  100.000000
2  _orig_mod.encoder.graph_layer_2.weight        183424    0.000000
3    _orig_mod.encoder.graph_layer_2.bias           128  100.000000
4      _orig_mod.encoder.decoder_1.weight       2053489    0.000097
5        _orig_mod.encoder.decoder_1.bias          1433    0.000000
6      _orig_mod.encoder.decoder_2.weight       2053489    0.000000
7        _orig_mod.encoder.decoder_2.bias          1433    0.000000
8         _orig_mod.encoder.linear.weight           896    0.000000
9           _orig_mod.encoder.linear.bias             7    0.000000
Saved sparsity snapshot to outputs_compiled/tran/cora/SAGE/seed_0/sparsity_after.csv
                               layer_name  num_elements  sparsity
0  _orig_mod.encoder.graph_layer_1.weight       2053489  0.000049
1    _orig_mod.encoder.graph_layer_1.bias          1433  0.000000
2  _orig_mod.encoder.graph_layer_2.weight        183424  0.000000
3    _orig_mod.encoder.graph_layer_2.bias           128  0.000000
4      _orig_mod.encoder.decoder_1.weight       2053489  0.000146
5        _orig_mod.encoder.decoder_1.bias          1433  0.000000
6      _orig_mod.encoder.decoder_2.weight       2053489  0.000097
7        _orig_mod.encoder.decoder_2.bias          1433  0.000000
8         _orig_mod.encoder.linear.weight           896  0.000000
9           _orig_mod.encoder.linear.bias             7  0.000000
Saved forward timing report to outputs_compiled/tran/cora/SAGE/seed_0/forward_times.csv
                         layer_name  avg_forward_time_ms  num_calls
0         _orig_mod.encoder.dropout             0.002243       3005
1   _orig_mod.encoder.graph_layer_1             0.002912       3005
2   _orig_mod.encoder.graph_layer_2             0.002368       3005
3       _orig_mod.encoder.decoder_1             0.002319       3005
4       _orig_mod.encoder.decoder_2             0.002276       3005
5          _orig_mod.encoder.linear             0.000564       3005
6   _orig_mod.encoder.vq.project_in             0.002233       3005
7  _orig_mod.encoder.vq.project_out             0.002526       3005
8    _orig_mod.encoder.vq._codebook             0.002895       3005
CPU Memory Usage: 1265.38 MB
Max GPU Memory Used: 619.52 MB
